webpack.config.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  //   optimization: {
+  //     runtimeChunk: 'single',
+  //   },
```

Comment:
Please can you remove this unnecessary comment because it is best practice to remove them to make your code clean?

Reply:
Done; Thanks :+1:

Reply:
- Resolve, well done :+1: